### ISSUE TITLE
Add back email validation step

### ISSUE THREAD

**Issue by jnordberg** [steemit/faucet#220]
Having the email validation step part of the approval is bad ux. Users expect a validation email. It also creates some issues that are hard to address without redesigning the system, namely:

  * Users can signup with a typo in their address, verify their number and never receive the approval email.
  * Users can register multiple accounts using the same email and different phone numbers

**Comment by gregory-latinier** [comment]
This was a request. See #157 
What do you want to do ?

**Comment by TimCliff** [comment]
I would recommend just starting with an email that gets sent after they have completed the step to verify their SMS, just letting them know that they are in the queue to be reviewed, and they will receive another email once the account is approved. This would fix about 80% of the UX issues. If you included the username they applied for, that would help too. You could also link to the steemit.com FAQ page where it recommends alternate signup methods (if they don't want to wait).

The corner case where they entered the wrong email is a more complicated issue. I suggest having a separate discussion on how to handle that in https://github.com/steemit/faucet/issues/205.

For the "multiple emails, same phone number" issue - can that just be solved on the approval end?

**Comment by gl2748** [comment]
Old email signup work:
https://github.com/steemit/faucet/pull/90

**Comment by Jolly-Pirate** [comment]
Email confirmation is important, because some users make typos in the account name and/or the email. It would be good to add a fields to retype the username and the email (not paste it) to be sure they don't mess things up.

**Comment by bnchdrff** [comment]
should email validation be required to proceed past the "Enter your email address" step of the form?

should we add a second "confirm your email" input on this step, also?

**Comment by Jolly-Pirate** [comment]
@bnchdrff I'm in favor of that, yes.

**Comment by TimCliff** [comment]
Actually adding email “verification” creates a complicated signup flow that I think more users will end up having an issue with. When they have to go from website, to email, then back to website in order to complete all the steps - there is a lot of room for error.

I suggest for this issue we just add an *informative* email, _after they confirm SMS_ to let the user know what their current status is (in the queue), provide them with their account name (so they have a record of it) and let them know we will send them another email once their account is approved.

Handing the case of an incorrect email being entered would be best addressed in another issue: https://github.com/steemit/faucet/issues/205
